src/locale/zh-cn.js

```diff
@@ -12,16 +12,16 @@ export default moment.defineLocale('zh-cn', {
     weekdaysShort : '周日_周一_周二_周三_周四_周五_周六'.split('_'),
     weekdaysMin : '日_一_二_三_四_五_六'.split('_'),
     longDateFormat : {
-        LT : 'Ah点mm分',
-        LTS : 'Ah点m分s秒',
-        L : 'YYYY-MM-DD',
+        LT : 'HH:mm',
+        LTS : 'HH:mm:ss',
+        L : 'YYYY年MMMD日',
         LL : 'YYYY年MMMD日',
-        LLL : 'YYYY年MMMD日Ah点mm分',
-        LLLL : 'YYYY年MMMD日ddddAh点mm分',
-        l : 'YYYY-MM-DD',
+        LLL : 'YYYY年MMMD日 HH:mm',
```

Comment:
@zenozeng Thx for following up! Would you mind giving a few examples that the new format of `LLL` and `LLLL` would cause inconvenience? 

If I understand correctly, the standard ([GB/T 15835-2011](https://github.com/moment/moment/files/599331/GB-T-15835-2011.pdf)) is for `when to use Chinese numerals vs. Arabic numbers` (一/二/三 vs. 1/2/3) in publications, not for `how to format time` in publications. This is shown in the preface of the standard, screenshot as below.

> This standard defines how to use Chinese numerals and Arabic numerals in publications.

![screenshot 2016-11-18 17 46 02](https://cloud.githubusercontent.com/assets/633094/20425759/0576f92e-adb7-11e6-8000-ccf4908d71e3.png)

To put it another way, I think the point of the standard is, when writing time like`4:12`, to enforce using `Arabic numerals` (4 and 12) instead of Chinese numbers (四 and 十二) in publication, not to enforce using `4时12分` over `4:12`.

Moreover, it actually suggests to use `GB/T7408-94 5.3.1.1 extended format when necessary`,  which users 24 hour format, like `04:00`, and is exactly what @jasonkb proposed here.